aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1 +1,360 @@
-More content coming soon.
+## Messaging Voice, Tone, and Content
+
+Depending on where you look, you will find different definitions for tone and voice.  By reading through many of these different definitions, a picture emerges:
+
+| **Voice**		                  | **Tone**                                    |
+| ----------------------------- | ------------------------------------------- |
+| Who they hear		              | What they hear                              |
+| Personality		                | Attitude                                    |
+| How you are doing the writing	| Who they hear                               |
+| Style		                      | How your message comes across               |
+| What makes us uniquely us		  | How we communicate to our specific audience |
+
+### Voice
+
+Voice is all about who the reader hears talking in our messaging; it has to do with the worldview of the author. Voice can be spoken, witty, friendly, formal, etc. We can be perceived as being intelligent, experienced, mature, accepting, objective, etc.
+
+A person's voice typically does not change once established. If our voice (HPE's voice) is inconsistent, people will notice. For example, people notice the difference between writing academically when compared to sounding like a 5-year old. This kind of inconsistency can result in lack of trust.
+
+### Tone
+
+Tone has to do with what is conveyed or how it comes across. It can express seriousness, urgency, playfulness, objectivity, intimacy, condescension, etc.
+
+Unlike voice, tone can change based on the target audience and the subject matter. It is situational. For example, tone is very different when giving directions during an emergency vs ordering a pizza.
+
```

Comment:
```suggestion

Because tone is situational, it may be tweaked to appropriately fit the context
within an HPE application. While tone may change, the message should be consistently
delivered in [HPE's voice](#voice) supporting the perception of intelligence, 
experience, objectivity, and inclusivity.
```